stock_warehouse_calendar_orderpoint/models/stock_warehouse.py

```diff
@@ -31,8 +31,30 @@ class StockWarehouse(models.Model):
         " Tuesday)",
     )
 
+    def get_company_from_ctx(self):
+        company = self.env.company
+        if self.env.context.get("force_wh_company"):
+            company = (
+                self.env["res.company"]
+                .browse(self.env.context["force_wh_company"])
+                .exists()
+            )
+        return company
+
     def _default_orderpoint_calendar_id(self):
-        return self.env.company.orderpoint_calendar_id
+        company = self.get_company_from_ctx()
+        return company.orderpoint_calendar_id
 
     def _default_orderpoint_on_workday_policy(self):
-        return self.env.company.orderpoint_on_workday_policy
+        company = self.get_company_from_ctx()
+        return company.orderpoint_on_workday_policy
+
+    @api.model
+    def create(self, vals):
+        """
+        We want to propagate the company_id in the case when we create a new company
+        and a corresponding WH is being created as a result.
+        """
```

Comment:
Better to not set docstrings on methods you override, as this is overriding the existing docstring.
Also, a docstring describes what the method does (here `create` create a record basically), the fact we play with the company is a technical detail for the user of this method. So a comment is more suitable here.

Reply:
@sebalix I changed the docstring to comment. Thank you for pointing this out.

Reply:
Maybe the commit message should be updated to fit OCA conventions:
- `[FIX] stock_warehouse_calendar_orderpoint: set proper company on warehouse`
or
- `stock_warehouse_calendar_orderpoint: set proper company on warehouse`

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message

Reply:
Good point, I corrected it.